mesonbuild/scripts/meson_install.py

```diff
@@ -168,6 +169,48 @@ def install_man(d):
         else:
             do_copyfile(full_source_filename, outfilename)
 
+def install_fs_links(d):
+    for source, to, type_ in d.fs_links:
+        to += os.path.splitext(source)[1]
+        print('Linking {} to {} as {} link"'.format(source, to, type_))
+
+        to = os.path.join(d.fullprefix, to)
+        append_to_log(to)
+        outdir = os.path.dirname(to)
+        if not os.path.isabs(source):
+            source = get_destdir_path(d, source)
```

Comment:
Hmm, doesn't this break with `DESTDIR`? Ideally, you want to do all this with relative paths so that the installed files can be transported to any prefix without breaking the links.

We should have a unit test that does that too. Installs the project, then moves it into a subdir and checks for broken symlinks.